script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt

```diff
@@ -891,7 +891,7 @@ Which means I'm―――
 -- アナタの<精神|トラウマ>、いただいちゃった♥」
 // トラウマ nonsensical enough on its own that I think this ruby deserves to stay
 // 精神 has to be mind here by decree of the lore council
-I've gobbled up that <mind|t r a u m a   > of yours ♥"
+I've gobbled up that <mind| t r a u m a   > of yours ♥"
```

Comment:
Is this intentional? If it's too left-justified, I'd delete from the padding at the end instead of adding padding at the front. _Should_ behave the same in-engine.